spec/controllers/proposals_controller_spec.rb

```diff
@@ -6,20 +6,19 @@
       {
         token: JWT.encode(
           { proposer_id: proposer.id },
-          Rails.application.credentials.jwt_secret!,
+          ENV.fetch('JWT_SECRET'),
           'HS512',
         ),
       }
     end
     let(:proposer) { users(:admin) }
     let(:proposee) { users(:user) }
 
-    context 'when a jwt_secret credential is available' do
-      before do
-        expect(Rails.application.credentials).
-          to receive(:jwt_secret!).
-          at_least(:once).
-          and_return('xyzdef2481632')
+    context 'when JWT_SECRET is set' do
+      around do |spec|
+        ClimateControl.modify(JWT_SECRET: 'zyx432') do
```

Comment:
Shortening the secret (vs `xyzdef2481632` before) makes it so that gitleaks doesn't pick this up as a "real" secret and warn about it.